File: Programmers/Sungwon/Level01/C++/x만큼간격이있는n개의숫자.cpp
//https://programmers.co.kr/learn/courses/30/lessons/12954
#include <string>
#include <vector>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> answer;
    int temp=x;
    for(int i=0; i<n; i++,temp+=x)
        answer.push_back(temp);
    return answer;
}

File: Programmers/Inho/C#/Level01/x만큼 간격이 있는 n개의 숫자.cs
using System;
public class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[] {};
        Array.Resize(ref answer, n);
        long now = x;
        
        for(int i=0;i<n;i++){
            answer[i] = now;
            now += x;
        }
        return answer;
    }
}